.gitignore

```diff
@@ -28,3 +28,6 @@ _testmain.go
 *.iml
 *.swp
 *.swo
+
+# vendor
+vendor
```

Comment:
Please revert this change...

Reply:
Any reason why this should be left included?

Reply:
Because this package doesn't have a `vendor` directory checked into git, so excluding it is actually no-op 😉 If you do anything with a vendor directory in your code, then it makes much more sense to exclude the directory in your own gitignore file.

Reply:
go mod vendor -> filled in the vendor dir -> need to actually check what files you want to add instead of all...